docs/contributing/contribution/development.md

```diff
@@ -0,0 +1,179 @@
+# Community Developer Guide
+
+```mermaid
+flowchart TD
+  check((Chat, Ask,<br>Investigate)) --> fork[Fork<br>the Project]
+  fork --> |code| code[Implement]
+  code --> |comment| done{Working?}
```

Comment:
Forking & Coding leads to implementation.

I don't fully understand what `comment` was intending here?

Reply:
The idea was to refer to journaling, but it's not clear. Like you implement something, and you do journaling about it. Maybe I should add a box there.

Reply:
Check 4316cde